### PR TITLE
Feat/add keyword focused

### PR DESCRIPTION
**Pull request by hawkingrei**
I add  a keyword `focused`.you can test by the following R code.
```R
settings <- function() {
    size(500, 500)
}

draw <- function() {
   stdout$print(processing$focused)
   stdout$print("\n")
}
```

### CODE REVIEW

File: src/rprocessing/RLangPApplet.java [hunk 1]
```diff
@@ -324,6 +324,26 @@ private boolean isMixMode() {
   }
 
   /**
+   *
```
Comment (gaocegege) [inline src/rprocessing/RLangPApplet.java]: Could you put these two functions in BuiltinApplet? I think the built-in function wrapper should be there

Comment (hawkingrei) [inline src/rprocessing/RLangPApplet.java]: 👊

Comment (hawkingrei) [inline src/rprocessing/RLangPApplet.java]: it is hard to move these two functions into `BuiltinApplet`

Comment (gaocegege) [inline src/rprocessing/RLangPApplet.java]: Got it.

**Comment by gaocegege** [comment]
LGTM

**Comment by jeremydouglass** [comment]
@hawkingrei -- thank you for contributing!

@gaocegege -- should we add an R version of the standard reference example -- https://processing.org/reference/focused.html ?

(BTW, shouldn't you be working on linuxcon right now??)

**Comment by gaocegege** [comment]
Yeah, I will add it after linuxcon. It is 23:00 in Shanghai, so it is break time 😄

**Comment by jeremydouglass** [comment]
I think the focused reference sketch was never added.

**Comment by gaocegege** [comment]
I have opened a new issue: #160

**Pull request merged**